WooCommerce/Classes/POS/Presentation/Settings/PointOfSaleSettingsView.swift

```diff
@@ -0,0 +1,223 @@
+import SwiftUI
+
+struct PointOfSaleSettingsView: View {
+    @Environment(\.dismiss) private var dismiss
+    @State private var selection: SidebarNavigation? = .store
+
+    var body: some View {
+        POSPageHeaderView(
+            title: Localization.navigationTitle,
+            trailingContent: {
+                Button(action: { dismiss() }) {
+                    Text(Image(systemName: "xmark"))
+                        .font(.posButtonSymbolLarge)
+                }
+                .foregroundColor(.posOnSurface)
+            })
+        HStack(spacing: POSSpacing.none) {
+            VStack(alignment: .leading, spacing: POSSpacing.none) {
+                List(selection: $selection) {
```

Comment:
Consider breaking these up into subviews... aim for something clearer at the top level like:

    var body: some View {
        POSPageHeaderView(
            ...
        )
        HStack {
            PointOfSaleSettingsList()
            PointOfSaleSettingsDetail()
        }
    }

You'll need to pass some bindings around but it'll be both easier to read/debug in future, and should perform better (not that there's a performance issue!)